src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -160,7 +159,6 @@ private function decideConsensus(TokenInterface $token, array $attributes, $obje
                     break;
 
                 default:
```

Comment:
Checked: there are more `default: break;` () constructs in components codebase, this is not the only one.